2021-11-01 21:17:05,788 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-02 17:19:15,523 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-02 17:20:56,213 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-02 17:21:21,348 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-02 17:39:39,510 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-02 18:20:19,462 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-03 12:56:21,889 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-03 12:56:49,457 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-03 13:01:36,911 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-03 13:05:09,004 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-03 13:06:31,697 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-03 13:07:20,254 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-03 13:09:23,335 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-03 13:10:03,809 ERROR: Exception on /explore [GET] [in /home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask/app.py:1457]
Traceback (most recent call last):
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/pazuzu71/microblog/app/routes.py", line 147, in explore
    return render_template('index.html', title='Explore', posts=posts)
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask/templating.py", line 147, in render_template
    return _render(
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask/templating.py", line 128, in _render
    rv = template.render(context)
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/pazuzu71/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pazuzu71/microblog/app/templates/base.html", line 31, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pazuzu71/microblog/app/templates/index.html", line 6, in block 'content'
    {{ form.hidden_tag() }}
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 474, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'form' is undefined
2021-11-03 13:10:22,223 ERROR: Exception on /explore [GET] [in /home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask/app.py:1457]
Traceback (most recent call last):
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/pazuzu71/microblog/app/routes.py", line 147, in explore
    return render_template('index.html', title='Explore', posts=posts)
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask/templating.py", line 147, in render_template
    return _render(
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/flask/templating.py", line 128, in _render
    rv = template.render(context)
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/pazuzu71/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pazuzu71/microblog/app/templates/base.html", line 31, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pazuzu71/microblog/app/templates/index.html", line 6, in block 'content'
    {{ form.hidden_tag() }}
  File "/home/pazuzu71/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 474, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'form' is undefined
2021-11-03 13:15:27,506 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-03 13:23:47,448 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-03 13:30:19,138 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-03 13:33:33,234 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:43]
2021-11-05 17:50:20,483 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:45]
2021-11-05 18:30:33,499 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:45]
2021-11-06 19:23:57,468 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:47]
2021-11-06 19:39:30,075 INFO: Microblog startup [in /home/pazuzu71/microblog/app/__init__.py:49]
